src/Model/Configuration/AbstractImportConfiguration.php

```diff
@@ -71,6 +73,23 @@ protected function prepareRecord(MatrixRecord $record, array $headerInfo): void
                     }
                 } elseif (!$locale) {
                     if (method_exists($entity, $setterName)) {
+                        
+                        $reflection = new \ReflectionMethod($entity, $setterName);
+                        $params = $reflection->getParameters();
```

Comment:
WHy this variable is needed?

Reply:
Yes, from now, it looks like it needs to check if it has some parameters, i think we could remove it be cause we have `if (isset($fieldDefinitions[$name]))` just after